### PR TITLE
Add Ict documentation

### PR DESCRIPTION
**Pull request by JakeSCahill**
We need a landing page for 'node software' where we can have links to IRI, Ict, and cIRI.

This is the start of the Ict docs that we can link to from that landing page.

### CODE REVIEW

**Comment by JakeSCahill** [comment]
Going to update the installation guide so that it doesn't use the script. Will also link to Lambtho's tutorial and the Dietpi image.

**Comment by JakeSCahill** [comment]
Have updated these docs in my local repo. I'm waiting on the IcTinder module to be released so i can finish the 'Find neighbors' article.

File: ict/0.1/introduction/ict-overview.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Ict overview
+
+**The Ict (IOTA Controlled agenT) is open-source Java software that defines an IOTA protocol, which is a lightweight alternative to the IRI. Computers that run the Ict are called Ict nodes.**
```
Comment (Citrullin) [inline ict/0.1/introduction/ict-overview.md]: I would clarify here. This sounds like ICT is part of the IOTA main tangle or dev tangle. As I understood it ICT is a completely different network with a different protocol and not integrated in any tangle maintained by IRI/cIRI.

Comment (Citrullin) [inline ict/0.1/introduction/ict-overview.md]: Do we want to differentiate between a node and an ICT node? And how is another implementation of the same concept called? Swarm node? Shouldn't we call it swarm nodes then?

Comment (JakeSCahill) [reply ict/0.1/introduction/ict-overview.md]: Hmm, true. I think I pulled that sentence from the IRI. No need for Ict here really 👍

**Comment by Citrullin** [comment]
@mikrohash Can you take a look into it?

**Comment by JakeSCahill** [comment]
Closing this until Bee is released and we can repurpose some of these articles

**Pull request closed without merging**